src/exodus/test/exodus_handler_tx.cpp

```diff
@@ -0,0 +1,191 @@
+#include "exodus/createtx.h"
+#include "exodus/errors.h"
+#include "exodus/encoding.h"
+#include "exodus/exodus.h"
+#include "exodus/wallettxs.h"
+
+#include "base58.h"
+#include "coins.h"
+#include "core_io.h"
+#include "main.h"
+#include "primitives/transaction.h"
+#include "script/script.h"
+#include "script/standard.h"
+#include "test/test_bitcoin.h"
+#include "test/fixtures.h"
+#include "utilstrencodings.h"
+
+#include "exodus/createpayload.h"
+#include "exodus/convert.h"
+
+#include "exodus/utilsbitcoin.h"
+
+#include "wallet/wallet.h"
+
+#include <boost/test/unit_test.hpp>
+
+#include <stdint.h>
+#include <string>
+#include <utility>
+#include <vector>
+
+/**
+ * Pushes bytes to the end of a vector.
+ */
+#define PUSH_BACK_BYTES(vector, value)\
+    vector.insert(vector.end(), reinterpret_cast<unsigned char *>(&(value)),\
+    reinterpret_cast<unsigned char *>(&(value)) + sizeof((value)));
+
+BOOST_FIXTURE_TEST_SUITE(exodus_handler_tx_tests, ZerocoinTestingSetup200)
+
+static std::vector<unsigned char> createMockSpendPayload()
+{
+    std::vector<unsigned char> payload;
+
+    uint16_t messageType = 1024;
+    uint16_t messageVer = 0;
+    exodus::swapByteOrder16(messageVer);
+    exodus::swapByteOrder16(messageType);
```

Comment:
Actually we don't need to swap bytes due to we support only little endian.

Reply:
nice catch

Reply:
after personal discussion, keep them.